templates/pioneer-study/esper-study-telemetry.1.parquetmr.txt

```diff
@@ -0,0 +1,65 @@
+message esper-study-telemetry {
+  required group metadata {
+    required int64  Timestamp;
+    required binary pioneerId (UTF8);
+    required binary studyName (UTF8);
+    optional binary reason (UTF8);
+    optional binary Date  (UTF8);
+    optional binary DNT (UTF8);
+    optional binary appUpdateChannel (UTF8);
+    required binary documentId (UTF8);
+    optional binary docType (UTF8);
+    optional binary os (UTF8);
+    optional binary sourceName (UTF8);
+    optional binary normalizedChannel (UTF8);
+    optional binary appVendor (UTF8);
+    optional binary clientId (UTF8);
+    optional binary Host (UTF8);
+    required binary geoCountry (UTF8);
```

Comment:
The geo metadata is a configuration option so they can be turned off.  Would you want every study submission to be flagged as an error in this case?

Reply:
@trink Yes, this particular study requires the geo metadata, so if the configuration option would be turned off, the resulting data set would not be useful.